docs/tutorials/data-based-matching.md

```diff
@@ -1,35 +1,40 @@
 ```
 torrent_name/
 ├─ torrent_file.mkv
 ```
+
 and separately match a single file torrent for `torrent_file.mkv`
 
-Due to using file system links, data based matching also opens the door for "risky" matching where the name of the searchee can be ignored and only file size is used, with a couple of caveats: 1) false positives are possible with this method, so it is recommended that you have cross-seed recheck torrents added during risky matching, and 2) risky matching only finds extra results for single-file searchees.
+Due to using file system links, data-based matching also opens the door for "risky" matching where the name of the searchee can be ignored and only file size is used, with a couple of caveats: 1) false positives are possible with this method, so it is recommended that you have cross-seed recheck torrents added during risky matching, and 2) risky matching only finds extra results for single-file searchees.
 
 ## Setup
 
-Data based matching adds a few parameters:
+Data-based matching adds a few parameters:
 
 `dataDirs`: This specifies what directories cross-seed should generate searchees from. They are specified as an array, for example `["/data/torrents/", "/data/media/"]` similar to the existing `torznab` parameter.
 
-`linkDir`: This specifies where you want your links cross-seed generates to be placed. While technically possible to specify the same directory as your dataDirs, this is not recommended and could result in bad interactions between existing and new files. Instead it should be a separate directory visible to both your torrent client and cross-seed.
+`linkDir`: This specifies where you want your links cross-seed generates to be placed. While technically possible to specify the same directory as your `dataDir`s, this is not recommended and could result in bad interactions between existing and new files. Instead, it should be a separate directory visible to both your torrent client and cross-seed.
 
-`linkType`: Either `hardlink` or `symlink`. We recommend symlinks, as if a source is removed for a hardlink, the new link still takes up space which is probably not desireable. If a symlink's source is removed, it's merely a broken symlink taking up basically no space on your file system. However, hardlinks are more flexible if you have a solution to this. For hardlinks, only the linkDir will need to be visible (with a matching path mapping) to your torrent client, whereas for symlinks both the linkDir and all specified dataDirs need to be visible with matching path mappings to your torrent client.
+`linkType`: Either `hardlink` or `symlink`. [(**See FAQ for more**)](http://localhost:3000/docs/reference/faq-troubleshooting#what-linktype-should-i-use-data-based-searching).
```

Comment:
fix link

Reply:
lol oops.

Reply:
this can be a relative link so it works both local and deployed